packages/orchestrator-ui-components/src/components/WFOPageTemplate/paths.ts

```diff
@@ -9,3 +9,4 @@ export const PATH_METADATA_RESOURCE_TYPES = '/metadata/resource-types';
 export const PATH_METADATA_WORKFLOWS = '/metadata/workflows';
 export const PATH_TASKS = '/tasks';
 export const PATH_SETTINGS = '/settings';
+export const PATH_SERVICE_TICKETS = '/service-tickets';
```

Comment:
After rebasing with main you will get a wfo-ui-surf app next to the standard app. You could add this entry overthere OR in the surf package you created in this PR